src/routes/casopisi/+page.svelte

```diff
@@ -1,15 +1,14 @@
 <script>
-	import magazines from './casopisi.json'
+	import magazinesInfo from './casopisi.json'
 
-	const {
-		title: firstMagTitle,
-		time: firstMagTime,
-		summary: firstMagSummary,
-		coverImageUrl: firstMagCoverImageUrl,
-		url: firstMagUrl
-	} = magazines[0]
+	const magazines = magazinesInfo.map((it) => ({
```

Comment:
pliz zamijeni `it` s `item` ili nečim sličnim, deskriptivnijim

Reply:
U redu, stavit ću `magazineInfo` kao naziv lambda argumenta.

P.S.
U slučajevima kada se koriste male lambda funkcije, obično se pridržavam Kotlinovog oblika imenovanja jednog lambda argumenta koristeći _[it](https://www.baeldung.com/kotlin/lambda-expressions#it)_. Sviđa mi se ta kratkoća, jednostavnost, a zatim i konzistentnost u argumentima malih lambda funkcija. :wink:

Reply:
moglo je ovdje biti i `mag` pa bi i dalje imao konciznost 😉 
`it` je jednako generično kao i dobri stari `i` i `j` u for petljama po studentski